src/components/checkWin.js

```diff
@@ -0,0 +1,21 @@
+export function checkWin(gameState, winningMessage, currentPlayerName) {
+  const winMessage = `${currentPlayerName} wins!`;
+
+  for (let i = 0; i < 3; i++) {
+    let rowSum = gameState[i][0] + gameState[i][1] + gameState[i][2];
```

Comment:
Add comments explaining what exactly happens here

Reply:
added: 
// checking fields values in rows for winning conditions